programmers/Ranking-of-lottos.js

```diff
@@ -0,0 +1,9 @@
+const win_nums = [31, 10, 45, 1, 6, 19];
+const lottos = [44, 1, 0 ,0, 31, 25];
+const Rankings = {6 : 1 , 5 : 2 , 4 : 3, 3 : 4, 2 : 5 , 1 : 6, 0 : 6};
+function solution(lottos, win_nums){
+    let Common = lottos.filter(x => win_nums.includes(x)).length;
+    let Zeros = lottos.filter(element => 0 === element).length;
+    return [Rankings[Common+Zeros],Rankings[Common]];
+}
```

Comment:
오브젝트를 이용하셔서 상당히 잘 푸셨네요
변수이름은 소문자로 시작합니다. 
const로 선언해도 되는것들은 const로 선언해주시는게 좋습니다.
```suggestion
function solution(lottos, win_nums) {
    const rankings = {6 : 1 , 5 : 2 , 4 : 3, 3 : 4, 2 : 5 , 1 : 6, 0 : 6};
    const common = lottos.filter(x => win_nums.includes(x)).length;
    const zeros = lottos.filter(element => 0 === element).length;
    return [rankings[common + zeros], rankings[common]];
}
```